### PR TITLE
add deploy/targets

### PR DESCRIPTION
**Pull request by flenter**
It may look like the text for the index has been complete removed, however it's just moved to steps.md

### CODE REVIEW

File: content/docs/deploy/steps.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+com/#applications/51c82a063179be4478002245/tab/details), deploying to
+[Cloud Foundry](https://app.wercker.com/#applications/53dd63d9df12aee638
+0c5b46/tab/details) or [AWS Code-Deploy](https://app.wercker.com/#applic
+ations/547fe5be6b3ba8733d2aabba/tab/details).
+
+Within deploy steps you can leverage
+[environment variables](/docs/environment-variables/index.html) for storing
+sensitive information such as passwords, IP addresses and tokens.
+
+Below a snippet for using the S3 deploy step:
```
Comment (mies) [inline content/docs/deploy/steps.md]: S3 synchronization

**Comment by mies** [comment]
small comment! Looks :100:

File: content/docs/deploy/index.md [hunk 2]
```diff
@@ -1,43 +1,29 @@
...
-        bucket-url: $AWS_BUCKET_URL
-        key-id: $AWS_ACCESS_KEY_ID
-        key-secret: $AWS_SECRET_ACCESS_KEY
-```
-
-Note that if you can't find a deploy step for your use-case you can
-[create your own!](/docs/steps/creating-steps.html)
+## Deploy
+
+A deploy is a [pipeline](/docs/pipelines/index.html) and similar to build can
```
Comment (Tonnu) [inline content/docs/deploy/index.md]: similar to a build pipeline

**Comment by Tonnu** [comment]
minor comment but looks :+1: :shipit:

**Comment by lindseybateman** [comment]
lgtm

**Pull request merged**